pontoon/base/static/css/translate.css

```diff
@@ -1445,19 +1445,26 @@ p#sign-in-required > a#sidebar-signin {
 }
 
 #helpers > section ul li > header button.approve:hover:before,
-#helpers > section ul li.translated > header button.approve:before {
+#helpers > section ul li.translated > header button.approve:before,
+#helpers > section ul li.translated > header button.unapprove:before {
   color: #7BC876;
   content: "";
 }
 
+#helpers > section ul li > header button.approve:before,
+#helpers > section ul li > header button.unapprove:hover:before {
+   content: "";
+   color: #4FC4F6;
+}
+
 #helpers > section ul li > header button.delete:before {
   content: "";
   margin-left: 5px;
 }
 
 #helpers > section ul li > header button.delete:hover:before {
   color: #FD5F60;
-}
+
```

Comment:
I'm sorry, it looks like a merge gone wrong between my laptop and my desktop. I'll fix this.